satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>What do you mean with a <code>pure</code>  image ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>